### ISSUE TITLE
Attempt to convert strings to date's causing some weird issues

### ISSUE THREAD

**Issue by kevindwells** [bguest/saneitized#1]
When passing in an array of domains (strings) the library is converting some of the domains to datetiems

``` ruby
t = ["five-nights-at-freddys-2.softonic.com",
      "five-nights-at-freddys-demo.en.softonic.com",
      "five-nights-at-freddys.en.softonic.com"]  
Saneitized.convert(t)
=> ["five-nights-at-freddys-2.softonic.com", 
      2015-08-20 17:00:00 -0400, 
      2015-08-20 17:00:00 -0400]
```

There are other cases where Chronic is erroring out trying to convert the string to a datetime

``` ruby
Saneitized.convert "cpr-first-aid.wonderhowto.com"
```

results in 

``` ruby
NoMethodError: undefined method `start=' for nil:NilClass
```

**Comment by bguest** [comment]
Noted: the date converter is rather aggressive, It should probably just be turned off by default. Not sure about the NoMethodError, I'll look into it.

**Comment by bguest** [comment]
fixed with v2.0.0
